modules/bwa/mem.nf

```diff
@@ -1,6 +1,6 @@
 process BWA_MEM {
     tag "${sampleName}"
-    label 'cpu_8_memory_8'
+    label 'cpu_8_memory_6'
```

Comment:
This change would mean instruct nextflow to put cpus=8 and memory=6 for all profiles i.e. AWS/docker etc.